### DISCUSSION TITLE
Pin Recommended Stackblitz for Repro Purposes in Issue Template

### DISCUSSION THREAD

**Discussion by jamaluddinrumi** [saadeghi/daisyui#2346] [Ideas / feature requests]
isn't it a good idea to pin this [stackblitz](https://stackblitz.com/edit/daisyui-cdn) as repro template in issue template? 

in hope everyone would providing repro url each creating an issue.

https://stackblitz.com/edit/daisyui-cdn

**Comment by saadeghi** [comment]
Good idea.  
However currently there's a Tailwind Play link and it's better to use that as a reproduction link because if something works on Tailwind Play and doesn't work on a framework (React, Vue, etc), it's not a daisyUI bug at all and shouldn't be an "issue" on this repo. It's defense either an incorrect config or it can be built tool bug. And in that case, it can be a discussion (but not an issue)
